infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/AbstractWorkspaceServiceAccount.java

```diff
@@ -133,6 +134,15 @@ private void createImplicitRolesWithBindings(Client k8sClient) {
         singletonList("metrics.k8s.io"),
         Arrays.asList("list", "get", "watch"),
         serviceAccountName + "-metrics");
+
+    // secrets role
+    createRoleWithBinding(
+        k8sClient,
+        SECRETS_ROLE_NAME,
+        Arrays.asList("secrets"),
+        singletonList(""),
+        Arrays.asList("list", "create", "delete"),
```

Comment:
is there a way to test these roles?

Reply:
I've added a testcase but all tests in this module are ignored. Should I create an issue for that?